helpers_test.go

```
@@ -0,0 +1,127 @@
+	fakeSmartctl := filepath.Join(tmpDir, "smartctl")
+	err := os.WriteFile(fakeSmartctl, []byte("#!/bin/sh\necho fake"), 0o755)
+	require.NoError(t, err)
+
+	// Temporarily prepend the fake path to the search list.
+	orig := smartctlSearchPaths
+	t.Cleanup(func() { smartctlSearchPaths = orig })
+	smartctlSearchPaths = append([]string{fakeSmartctl}, orig...)
+
+	// Clear PATH so exec.LookPath cannot find anything.
+	origPath := os.Getenv("PATH")
+	t.Cleanup(func() { os.Setenv("PATH", origPath) })
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)